src/System.Linq/samples/System.Linq.Samples.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{D8207664-0C98-4DC9-8249-02546D21C7A4}</ProjectGuid>
+    <OutputType>Library</OutputType>
```

Comment:
You're probably aware, but this means the project can't actually be run as a console application. We do build a few console applications targeting the CoreCLR surface area in buildtools.